src/app/blog/index.html

```diff
@@ -5,6 +5,18 @@
 	showTableOfContents="true">
 	
 	<article class="blog-post">
+		<header>
+			<stache-page-anchor>
+				Design update — April 2020
+			</stache-page-anchor>
+		</header>
+		<section>
+      <h3 class="sky-subsection-heading">Recently completed</h3>
+        <p>There were no new components this month as we spent time fixing bugs and refactoring some component code.</p>
```

Comment:
It's minor, but I'd tweak the start of this to use active voice: "We didn't add any new components..."

No big deal if you don't want to make this change though, so I'll go ahead and approve. ... But it looks like we have a bigger problem in that one of the checks is hanging. Let me know if you want my help to dig into what's happening with that.

Reply:
updated